.github/workflows/continuous-integration.yml

```diff
@@ -35,18 +35,20 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - name: Update NPM
+        if: matrix.node-version <= 14
         run: npm install -g npm
       - name: Bootstrap project
         run: |
           npm ci --ignore-scripts
           npm run postinstall
       - name: Build project
-        run: |
-          npm run clean
-          npm run build
-      - uses: Yuri6037/Action-FakeTTY@v1.1
+        run: npm run build
       - name: Run tests
-        run: faketty npm test --ignore-scripts
+        run: node packages/build/bin/run-nyc node packages/build/bin/run-mocha --lang en_US.UTF-8 --reporter spec "packages/*/dist/__tests__/**/*.js" "extensions/*/dist/__tests__/**/*.js" "examples/*/dist/__tests__/**/*.js" "packages/build/test/*/*.js"
+      - name: Run CLI tests
+        if: runner.os == 'Linux'
+        shell: 'script --return --quiet --command "bash {0}"'
+        run: node packages/build/bin/run-nyc node packages/build/bin/run-mocha --lang en_US.UTF-8 --reporter spec "packages/cli/test/**/*.js"
```

Comment:
Can we keep CLI testing (through faketty) for macOS? Otherwise, we can't guarantee that the CLI functions on that environment.

Reply:
Yes. As soon as we can, we should remove it. It leaves us blind to any problem.